website/src/plugins/remark-internal-links.ts

```
@@ -0,0 +1,43 @@
+
+  const hashIndex = node.url.indexOf('#');
+  const hash = hashIndex === -1 ? '' : node.url.slice(hashIndex);
+  let path = (hashIndex === -1 ? node.url : node.url.slice(0, hashIndex))
+    // Typedoc links to the source files it generated.
+    .replace(markdownExtensionRe, '');
+
+  // Leave assets (/img/logo.png) alone — only page routes are directories.
+  if (fileExtensionRe.test(path)) return;
+
+  // Typedoc keeps the declaration's casing, but routes are built from
+  // lowercased slugs and GitHub Pages is case-sensitive.
```

ESLint

[error] Expected a block comment instead of consecutive line comments.

(multiline-comment-style)